### ISSUE TITLE
Exemplar images on novel classes

### ISSUE THREAD

**Issue by Lukas-Ma1** [Zehong-Ma/OVMR#4]
Hi! What a impressive work on open-vocabulary field! Still I have a question about your operation on novel classes.

In your implemention on multi-modal classifier, to determine whether a picture belongs to the dog category, you need:
 1. class label, 2. some examplar images of dog, which are gathered in your pre-process steps. 

However, the setting of ov classification requires invisibility to novel categories of images. How do you gather the novel images in order to accomplish multi-modal classifier, if you strictly follow the open-vocabulary setting. Are you filter the novel images through several operations, or directly select from datasets randomly?

**Comment by Zehong-Ma** [comment]
When evaluating on novel classes, the exemplar images are sampled from the corresponding training set of these novel classes. We sampled the images for novel classes from the existing annotated training set. In our paper, users should provide the exemplar images of the classes that they are interested in.  We remove all categories of the evaluation datasets from our pre-training data ImageNet21k-OVR. So the categories in the evaluation datasets are "novel". Users should select exemplar images for each target category according to their comprehension of the concept.  It is worth noting that both textual descriptions and visual exemplars of the target class are clues to define a better classifier. Users should define what they want to recognize clearly! Our method provides a powerful and effective way to inject and fuse multi-modal clues.

**Comment by Lukas-Ma1** [comment]
Appreciated your patient explain! It seems that your approach is a little bit like few-shot methods. The information of novel classes didn't appear in pre-training, but appear in training process.

**Comment by Zehong-Ma** [comment]
Apologies for the confusion. In our experiments, the classes used in evaluation are completely excluded from the pre-training phase. When evaluated on 11 downstream classification tasks, OVMR simply **forwards the exemplar images to generate a multi-modal classifier, without requiring any additional training**. In contrast, prompt learning and few-shot learning methods do require further training. For clarity, please refer to Sec. 4.1 Datasets (**Pre-training Set**) and Sec. 4.2 Implementation Details (**Training and Test Setup**). Consequently, information about the novel classes is absent in both the pre-training phase and downstream fine-tuning.

**Comment by Zehong-Ma** [comment]
Apologies for the confusion. In our experiments, the classes used in evaluation are completely excluded from the pre-training phase. When evaluated on 11 downstream classification tasks, **OVMR simply forwards the exemplar images to generate a multi-modal classifier, without requiring any additional training**. In contrast, prompt learning and few-shot learning methods do require further training. For clarity, please refer to Sec. 4.1 Datasets (**Pre-training Set**) and Sec. 4.2 Implementation Details (**Training and Test Setup**). Consequently, information about the novel classes is absent in both the pre-training and downstream fine-tuning in our method. Our method is training-free and plug-and-play.

**Comment by Lukas-Ma1** [comment]
I see, thanks for your help and good luck to your cvpr 25’！
